var/spack/repos/builtin/packages/proj/package.py

```diff
@@ -89,6 +89,7 @@ class Proj(CMakePackage, AutotoolsPackage):
 
     patch("proj.cmakelists.5.0.patch", when="@5.0")
     patch("proj.cmakelists.5.1.patch", when="@5.1:5.2")
+    patch("proj-8.1-cmake-3.29-new-tiff-interface.patch", when="+tiff @:9.1.0 ^cmake@3.19:")
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
It seems like the 3.28 PR is failing on this as well, I'm going to make this its own PR and apply suggested changes there. Will tag you (and remove this change here) once that's up.